### PR TITLE
added WINDOWS troubleshoots and solutions for setting up environment

### PR DESCRIPTION
**Pull request by suyetake**
This PR closes #[insert associated issue number here]
some Windows woes when installing node versions for yarn
## What does this PR do?
adds a WINDOWS section and info to the readme install notes
## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)
https://gph.is/YBr328

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,37 @@ To fix this, run the script:
 yarn build:local-packages
 ```
 
+#### WINDOWS Installation Notes
+
+ scoop tool installation:	// involves some powershell env setups
+  https://github.com/lukesampson/scoop/wiki/Quick-Start
```
Comment (jacobvenable) [inline README.md:106]: The problem you faced with setting up in Windows could also happen on other operating systems. What do you think about restructuring this to something like:

```
#### Incorrect Node Versions

To match our production environment, we specify our local environment to use a specific version of Node. If you don't already have that version, switching Node versions can be a hassle.

##### OSX

There are various CLIs that can assist with easily switching between versions of Node

- [NVM](https://github.com/nvm-sh/nvm/blob/master/README.md)
- [n](https://github.com/tj/n)

##### Windows

- [NVM for Windows](https://github.com/coreybutler/nvm-windows)
- [Scoop](https://github.com/lukesampson/scoop/wiki/Quick-Start)
```

File: README.md [hunk 2]
```diff
@@ -100,6 +100,37 @@ To fix this, run the script:
...
+
+```
+ scoop install -g yarn
+ scoop install -g nodejs
+ yarn --version			// 1.22.4 in May/June 2020
+ node --version			// v14.4.0 on June 3, 2020
+```
```
Comment (jacobvenable) [inline README.md:113]: Were there directions for installing `yarn` and `nodejs` elsewhere we could link to?

Comment (rhinodavid) [reply README.md:113]: 2cents: We could put links to the package.json here instead of updating the readme with every version bump

**Comment by jacobvenable** [comment]
Going to close this for now but please feel free to re-open once you've had a chance to finish. Thanks!

**Pull request closed without merging**